acceptance/setup/common/00_EnvSetup.rb

```diff
@@ -46,5 +46,11 @@
   when /solaris/
     step "#{host} Install json from rubygems"
     on host, 'gem install json'
+
+    step "#{host} Install deep_merge from rubygems"
+    on host, 'gem install deep_merge'
+  else
+    step "#{host} Install deep_merge from rubygems"
+    on host, 'gem install deep_merge'
```

Comment:
I don't think we're running Solaris acceptance of hiera right now. But when we do it should be with aio packages of solaris.